src/views/partials/DeployApplication/EnvironmentVariablesEditor.vue

```diff
@@ -35,10 +35,23 @@ const props = defineProps({
 })
 
 const environmentVariablesKeys = toRef(props, 'environmentVariablesKeys')
+
+const handlePaste = (event) => {
+  const oldLength = props.environmentVariablesKeys.length
+  const clipboardText = event.clipboardData.getData('text')
+  const lines = clipboardText.split('\n')
```

Comment:
filter out blank lines from list as well, then it should be fine.
because sometimes we put blank line at the end of .env content

Reply:
yeah i'll do that too

Reply:
I've pushed one with validations